server/routes/deleteAddress.ts

```diff
@@ -29,12 +29,13 @@ const deleteAddress = async (models, req: Request, res: Response, next: NextFunc
     return next(new Error(Errors.AddressNotFound));
   }
 
-  if (req.body.chain) {
-    const existingMemberships = await models.Membership.findAll({
-      where: { chain: req.body.chain, user_id: req.user.id, active: true }
-    });
-    if (existingMemberships.length > 0) return next(new Error(Errors.LastAddress));
-  }
+  // TODO: Membership Membership removed
```

Comment:
What's this TODO for?

Reply:
I think we removed memberships in `raymond.construct` and so these models don't exist anymore. This is probably good to delete in that case.